packages/meditrak-server/src/app/addRoutesToApp.js

```diff
@@ -142,13 +158,22 @@ export function addRoutesToApp(app) {
   app.get('(/v[0-9]+)/permissionGroups/:recordId?', getPermissionGroups);
   app.get('(/v[0-9]+)/options/:recordId?', getOptions);
   app.get('(/v[0-9]+)/optionSets/:recordId?', getOptionSets);
+  app.get('(/v[0-9]+)/optionSets/:parentRecordId/options', getOptions);
   app.get('(/v[0-9]+)/users/:recordId?', getUserAccounts);
   app.get('(/v[0-9]+)/userEntityPermissions/:recordId?', getUserEntityPermissions);
   app.get(
     '(/v[0-9]+)/users/:parentRecordId/userEntityPermissions/:recordId?',
     getUserEntityPermissions,
   );
   app.get('(/v[0-9]+)/accessRequests/:recordId?', getAccessRequests);
+  app.get('(/v[0-9]+)/data_sources/:recordId?', getDataSources);
+  app.get('(/v[0-9]+)/data_sources/:parentRecordId/data_sources', getDataSources);
```

Comment:
Can you make these consistent, even if it means making changes in the admin panel code too? The endpoints won't be hit by any other clients at this stage, so that's safe.